src/query/algorithms/triangles/StreamingTriangles.cpp

```diff
@@ -60,27 +62,78 @@ NativeStoreTriangleResult StreamingTriangles::countLocalStreamingTriangles(
     return nativeStoreTriangleResult;
 }
 
-std::string StreamingTriangles::countCentralStoreStreamingTriangles(
-        std::vector<JasmineGraphIncrementalLocalStore*> incrementalLocalStoreInstances) {
+std::map<long, std::unordered_set<long>> StreamingTriangles::getCentralAdjacencyList(unsigned int graphID,
+                                                                                     unsigned int partitionID) {
+    std::map<long, std::unordered_set<long>> adjacencyList;
+
+    GraphConfig gc;
+    gc.graphID = graphID;
+    gc.partitionID = partitionID;
+    gc.maxLabelSize = 43;
```

Comment:
Can we set the 43 into a constant? why this is 43?

Reply:
done

Reply:
I have put some more comments, please address them.

Reply:
Ok